### PR TITLE
Added ability to download geometry from github branches

### PR DESCRIPTION
**Pull request by Yug34**

### CODE REVIEW

**Comment by Yug34** [comment]
Ah well, just realized if you need more than a single geometry file, this wouldn't be useful.

File: job.sh [hunk 1]
```diff
@@ -3,7 +3,11 @@
 source /etc/profile
 source /jlab/remoll/bin/remoll.sh
 
+#curl https://github.com/$3/remoll/blob/$4/geometry/$5 -o $5
+
```
Comment (wdconinc) [inline job.sh]: I think we should use git clone but only if the argument is non-zero, so
```suggestion
if [ -n "$3" ] ; then
  git clone --depth 1 --branch ${3/*:/} https://github.com/${3/:*/} remoll
  ln -sf remoll/geometry geometry
fi
```

**Comment by wdconinc** [comment]
Let me know when this is ready for merging.

**Comment by Yug34** [comment]
Addressed #8 and #9

**Comment by wdconinc** [comment]
Ready for merging?

**Comment by Yug34** [comment]
Yup, ready for merging!

**Pull request merged**